### DISCUSSION TITLE
Disturbing horizontal scroll bar on contributions calendar

### DISCUSSION THREAD

**Discussion by gruvw** [community/community#73117] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

I have recently noticed the appearance of a scroll bar on contributions calendar. I is really disturbing and non-accessible.  
I think it's because you moved the "dates" filters to the side of the contribution calendar.

![image](https://github.com/community/community/assets/63407038/5bbb01da-f4cd-436f-8771-06980900203c)

### Expected behavior

At least start the scroll from the right so we can see more recent contributions right away.

But there should not be any horizontal scroll bar here in the first place... either reduce the size of the overall calendar, put back the dates filter under it, crop the calendar to not display older weeks.

### Infos

- OS: Ubuntu 22.04.3 LTS x86_64
- Browser: Brave Browser 117.1.58.135

**Comment by LiilBoo** [comment]
Hi, 

yes I came here for the same reason

I have experimented the same issue.And I agree with all the suggestions proposed by gruvw

**Comment by onemoreangle** [comment]
Noticed this too starting at the end of last week, maybe it had something to do with Halloween's change? But makes it even more strange that this hasn't been addressed

**Comment by ricardonunesdev** [comment]
+1

Same issue here, when logged in the years selector is beside the contribution graph, which now has a horizontal scrollbar. When logged out it's fine.

**Comment by BryanNikla** [comment]
+1 

Same issue, first appeared alongside the Halloween colors.

**Comment by gruvw** [comment, marked as answer]
It has been fix for me. Thank you :)
